package.json

```diff
@@ -43,6 +43,9 @@
     "lodash.groupby": "^4.6.0",
     "semver": "^5.4.1"
   },
+  "peerDependencies": {
+    "webpack": "^4.30.0 || ^5.0.0"
```

Comment:
Normally it is non-passive to add a peer dependency, but this dependency was previously implied. How else are you going to use a webpack plugin? Now it's explicit.